server/app/controllers/api/ApiDocsController.java

```diff
@@ -48,16 +48,18 @@ public Result docsForSlug(Http.Request request, String programSlug) {
 
     ImmutableSet<String> allProgramSlugs = programService.getAllProgramSlugs();
 
+    ProgramDefinition programDefinition;
     try {
-      ProgramDefinition programDefinition =
+      programDefinition =
           programService.getActiveProgramDefinitionAsync(programSlug).toCompletableFuture().join();
-      return ok(docsView.render(request, programDefinition, allProgramSlugs));
     } catch (Exception e) {
       return notFound(
           String.format(
               "No active programs found for %s. Please create and publish a program with this slug"
                   + " to continue.",
               programSlug));
     }
+
+    return ok(docsView.render(request, programDefinition, allProgramSlugs));
   }
```

Comment:
Earlier I ran into an exception in the page rendering that said "No active programs found..." so I moved the `return ok(...)` outside the try so that the exception will be more clear if it were to happen again.